src/ctirs/api/v1/gv/views.py

```diff
@@ -873,16 +730,9 @@ def stix_files_id(request, package_id):
 
 
 # GET /api/v1/gv/stix_file/<package_id>
-# stix_file 返却
 def get_stix_files_id(request, package_id):
```

Comment:
No decorator.

Reply:
`get_stix_files_id` calls from `stix_files_id` only.
`stix_files_id` has `@api_key_auth`.
So authentication is complete at this stage.